contracts/Cash.sol

```diff
@@ -50,4 +56,48 @@ contract Cash is ERC20Burnable, Operator {
     {
         super.burnFrom(account, amount);
     }
+    
+    function _transfer(address sender, address recipient, uint256 amount) 
+        internal 
+        override 
+    {
+        require(sender != address(0), "ERC20: transfer from the zero address");
+        require(recipient != address(0), "ERC20: transfer to the zero address");
+
+        if (amount > 0) {
+            _beforeTokenTransfer(sender, recipient, amount);
+            _balances[sender] = _balances[sender].sub(amount, "ERC20: transfer amount exceeds balance");
+
+            if(feeChecker.isTransferTaxed(sender, recipient)) {
+                uint256 feeAmount = IFeeChecker(feeChecker).calculateFeeAmount(sender, recipient, amount);
+                uint256 transferToAmount = amount.sub(feeAmount);
+
+                require(transferToAmount.add(feeAmount) == amount, "transferToAmount + feeAmount != amount");
```

Comment:
This require only checks if the SafeMath library functions add and sub are cancelling each other out.
I would omit it, we can have range checks in the feeChecker and the SafeMath library already protects from overflow.

Reply:
This check has been removed